daemon/mgr/volume.go

```diff
@@ -147,6 +157,26 @@ func (vm *VolumeManager) Attach(ctx context.Context, name string, options map[st
 	id := types.VolumeID{
 		Name: name,
 	}
+
+	v, err := vm.Get(ctx, name)
+	if err != nil {
+		return nil, errors.Errorf("failed to get volume: %s", name)
+	}
+
+	if options == nil {
+		options = make(map[string]string)
+	}
+
+	cid, ok := options[types.OptionRef]
+	if ok && len(cid) != 0 {
```

Comment:
cid == "" will be better WDYT???

Reply:
@HusterWan I think two kinds are the same. Just keep the consistent in one project.